app/jobs/api/issues_open_issue_job.rb

```diff
@@ -6,6 +6,7 @@ class IssuesOpenIssueJob < IssueEventJob
 
     def payload(params)
       {
+        pull_request: params[:pull_request],
```

Comment:
I'm adding `:pull_request` to the base payload, not attaching it to the `:issue` hash.